pre-process-less/index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Riot Example: Pre-process LESS</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <message-one></message-one>
+  <message-two></message-two>
+
+  <!-- riot tags -->
+  <script src="message-one.tag" type="riot/tag"></script>
+  <script src="message-two.tag" type="riot/tag"></script>
+
+  <!-- scripts we need -->
+  <script src="https://cdn.jsdelivr.net/less/2.2.0/less.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/g/riot@2.2(riot.min.js+compiler.min.js)"></script>
+
+  <!-- mount this app -->
+  <script>
+    riot.parsers.css.less = function(tagName, css) {
```

Comment:
Sorry, a small point. The parameter `css` seems containing LESS...? Would it be more readable to change?
- `css` --> `style`
- `style` --> `css`